apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.ts

```diff
@@ -9,11 +9,11 @@ import {
   Group,
   Type,
 } from '@perun-web-apps/perun/openapi';
-import { createNewApplicationFormItem } from '@perun-web-apps/perun/utils';
+import { createNewApplicationFormItem, escapeDangerousHtml } from '@perun-web-apps/perun/utils';
 import DisabledEnum = ApplicationFormItem.DisabledEnum;
 import HiddenEnum = ApplicationFormItem.HiddenEnum;
 import { ItemType, NO_FORM_ITEM, SelectionItem } from '@perun-web-apps/perun/components';
-import { StoreService } from '@perun-web-apps/perun/services';
+import { AuthService, StoreService } from '@perun-web-apps/perun/services';
```

Comment:
unused import

Reply:
Good point; I must have overlooked it. The more I think about it, I should combine the DOMPurify and the specific property filter (coded in this PR).